codecov.yml

```diff
@@ -0,0 +1,15 @@
+# Always comment on PRs
+comment:
+  layout: "diff, flags, files" # delete old and post new
+  behavior: new
+  require_changes: false
+
+coverage:
+  status:
+    project:
+      default:
+        target: 80%
+        threshold: 1%
+
```

Comment:
we should add:
```
patch:
      default:
        target: 80%
        threshold: 1%
```

So that every PR has at least 80% coverage. not sure if setting the needed coverage at the project level will also set it as requirement at the patch (PR) level.

Reply:
You're right, good catch.